examples/manoeuvring.html

```diff
@@ -0,0 +1,393 @@
+<html>
+	<head>
+		<title>Ship in Ocean</title>
+		<script src="../build/vessel.js"></script>
+
+		<script src="libs/three_r118.js"></script>
+		<script src="libs/STLLoader.js"></script>
+		<script src="libs/GLTFLoader.js"></script>
+
+		<script src="snippets/Ship3D_v2.js"></script>
+
+		<!-- Upgrading to WaterShader2.js will remove the dependency
+	on Mirror.js as well as open up possibilities for visualizing approximate water flows around vessels. -->
+		<!-- <script src="libs/Mirror.js"></script>-->
+		<!-- <script src="libs/WaterShader.js"></script> -->
+		<script src="libs/Water.js"></script>
+
+		<!-- Bootstrap Core CSS -->
+		<link href="examples/libs/bootstrap.min.css" rel="stylesheet">
+
+		<!-- jQuery Version 1.11.1 -->
+		<script src="examples/libs/jquery.js"></script>
+
+		<script src="libs/OrbitControls.js"></script>
```

Comment:
Use the latest version OrbitControls_v2.js I am reorganizing the files right now.